package com.ultracar.desafio_ultracar.exceptions;

public class DataAgendamentoInvalidaException extends RuntimeException {
  public DataAgendamentoInvalidaException(String message) {
    super(message);
  }
}
